src/university/universityParts/University.java

```diff
@@ -0,0 +1,85 @@
+package university.universityParts;
+
+import university.exceptions.nullListExceptions.NullFacultyListException;
+
+import java.text.DecimalFormat;
+
+/**
+ * University is a singletone class, as the program is written for a specific university.
+ * It contains the list of faculties and the list of students in the university, as well
+ * some methods to work with those lists.
+ *
+ * @author Karine Gevorgyan
+ */
+
+public class University {
+
+    final static DecimalFormat numberFormat = new DecimalFormat("#.0");
+    private String name;
+    private Faculty[] faculties;
+    private String[] facultyNames;
```

Comment:
Karine jan don't store  facultyNames in class if you calculate them each time with getFacultyNames(..) method